sql/moz-fx-data-shared-prod/telemetry/active_users/view.sql

```diff
@@ -0,0 +1,193 @@
+  CAST(mozfun.bits28.days_since_seen(days_seen_bits) < 7 AS BOOLEAN) AS is_weekly_user,
+  CAST(mozfun.bits28.days_since_seen(days_seen_bits) < 28 AS BOOLEAN) AS is_monthly_user,
+  TRUE AS is_desktop,
+  FALSE AS is_mobile
+FROM
+  `moz-fx-data-shared-prod.klar_android.baseline_clients_last_seen`
+UNION ALL
+-- Klar iOS
+SELECT
+  submission_date,
+  client_id,
+  CASE
+    WHEN isp = 'BrowserStack'
+      THEN CONCAT('Klar iOS', ' ', isp)
+    ELSE 'Klar iOS'
+  END AS app_name,
+  days_seen_bits,
+  days_active_bits,
+  CAST(mozfun.bits28.days_since_seen(days_active_bits) = 0 AS BOOLEAN) AS is_dau,
+  CAST(mozfun.bits28.days_since_seen(days_active_bits) < 7 AS BOOLEAN) AS is_wau,
+  CAST(mozfun.bits28.days_since_seen(days_active_bits) < 28 AS BOOLEAN) AS is_mau,
+  CAST(mozfun.bits28.days_since_seen(days_seen_bits) = 0 AS BOOLEAN) AS is_daily_user,
+  CAST(mozfun.bits28.days_since_seen(days_seen_bits) < 7 AS BOOLEAN) AS is_weekly_user,
+  CAST(mozfun.bits28.days_since_seen(days_seen_bits) < 28 AS BOOLEAN) AS is_monthly_user,
+  FALSE AS is_desktop,
+  FALSE AS is_mobile
+FROM
+  `moz-fx-data-shared-prod.klar_ios.baseline_clients_last_seen`
+UNION ALL
+-- Focus Android
+SELECT
+  submission_date,
+  client_id,
+  CASE
+    WHEN isp = 'BrowserStack'
+      THEN CONCAT('Focus Android', ' ', isp)
+    ELSE 'Focus Android'
+  END AS app_name,
+  days_seen_bits,
+  days_active_bits,
+  CAST(mozfun.bits28.days_since_seen(days_active_bits) = 0 AS BOOLEAN) AS is_dau,
+  CAST(mozfun.bits28.days_since_seen(days_active_bits) < 7 AS BOOLEAN) AS is_wau,
+  CAST(mozfun.bits28.days_since_seen(days_active_bits) < 28 AS BOOLEAN) AS is_mau,
+  CAST(mozfun.bits28.days_since_seen(days_seen_bits) = 0 AS BOOLEAN) AS is_daily_user,
+  CAST(mozfun.bits28.days_since_seen(days_seen_bits) < 7 AS BOOLEAN) AS is_weekly_user,
+  CAST(mozfun.bits28.days_since_seen(days_seen_bits) < 28 AS BOOLEAN) AS is_monthly_user,
+  FALSE AS is_desktop,
+  IF(LOWER(isp) <> "browserstack", TRUE, FALSE) AS is_mobile
+FROM
+  `moz-fx-data-shared-prod.focus_android.baseline_clients_last_seen`
+UNION ALL
+-- Focus iOS
+SELECT
+  submission_date,
+  client_id,
+  CASE
+    WHEN isp = 'BrowserStack'
+      THEN CONCAT('Focus iOS', ' ', isp)
+    ELSE 'Focus iOS'
+  END AS app_name,
+  days_seen_bits,
+  days_active_bits,
+  CAST(mozfun.bits28.days_since_seen(days_active_bits) = 0 AS BOOLEAN) AS is_dau,
+  CAST(mozfun.bits28.days_since_seen(days_active_bits) < 7 AS BOOLEAN) AS is_wau,
+  CAST(mozfun.bits28.days_since_seen(days_active_bits) < 28 AS BOOLEAN) AS is_mau,
+  CAST(mozfun.bits28.days_since_seen(days_seen_bits) = 0 AS BOOLEAN) AS is_daily_user,
+  CAST(mozfun.bits28.days_since_seen(days_seen_bits) < 7 AS BOOLEAN) AS is_weekly_user,
+  CAST(mozfun.bits28.days_since_seen(days_seen_bits) < 28 AS BOOLEAN) AS is_monthly_user,
+  FALSE AS is_desktop,
+  IF(LOWER(isp) <> "browserstack", TRUE, FALSE) AS is_mobile
```

Comment:
```suggestion
  IF(LOWER(COALESCE(isp, "")) <> "browserstack", TRUE, FALSE) AS is_mobile
```
^ This change needs to be made to all of the `isps` to correctly handle the NULL case.